org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/org.OpenT2T.Sample.SuperPopular.Thermostat-2.0.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+    <description>OpenT2T SuperPopular Thermostat interface</description>
+
+    <property name="PowerStatus" type="s" access="readwrite">
```

Comment:
Oh, you're right. In that case, we'll need some code in DeviceAccessor to un-capitalize the names when looking for corresponding JavaScript translator properties, when the translator uses properties instead of accessor methods.

---
In reply to: [73771185](https://github.com/openT2T/translators/pull/37#discussion_r73771185) [](ancestors = 73771185,73768119)